tests/pytests/integration/test_minion.py

```diff
@@ -0,0 +1,52 @@
+import time
```

Comment:
Let's instead add this test module under `tests/pytests/integration/minion/test_return_retries.py`

Reply:
Or publish retries, whichever makes more sense.

Reply:
Sure thing, moved it there. I've kept the `test_return_retries` name as it's generally named as `return` in docs/code.